receiver/gitproviderreceiver/internal/scraper.go

```diff
@@ -17,8 +17,6 @@ type ScraperFactory interface {
 	CreateMetricsScraper(ctx context.Context, params receiver.CreateSettings, cfg Config) (scraperhelper.Scraper, error)
 }
 
-type Config interface {
-}
+type Config interface{}
 
-type ScraperConfig struct {
-}
+type ScraperConfig struct{}
```

Comment:
either revert the whitespace changes or make them separate from this PR for easier review